Jenkinsfile-deploy

```diff
@@ -56,11 +64,17 @@ pipeline {
 }
 
 def getAccountNumberFromStage() {
+
+    def accountNumber = env.INTG_ACCOUNT
+
     def stageToAccountMap = [
-            "intg": env.INTG_ACCOUNT,
-            "staging": env.STAGING_ACCOUNT,
-            "prod": env.PROD_ACCOUNT
+        "staging": env.STAGING_ACCOUNT,
+        "prod": env.PROD_ACCOUNT
     ]
 
-    return stageToAccountMap.get(params.STAGE)
+    if (stageToAccountMap.get(params.STAGE)) {
+        accountNumber = stageToAccountMap.get(params.STAGE)
+    }
+
+    return accountNumber
```

Comment:
Is it possible to use the Java map method `.getOrDefault` in Groovy to simplify this?

Also, do we want a default? It seems like it might be better for the job to fail if the environment is misconfigured, rather than assuming it means integration.

Reply:
Good point about it should be failing if the environment is misconfigured